Pod/Public/TCCAnimationTileOverlayRenderer.m

```diff
@@ -42,11 +45,11 @@ - (int)tileCoordinateSizeForZoomLevel:(int)zoomLevel {
 
 - (BOOL)canDrawMapRect:(MKMapRect)mapRect zoomScale:(MKZoomScale)zoomScale
 {
-    self.renderedTileZoomLevel = [TCCMapKitHelpers zoomLevelForZoomScale:zoomScale];
-    
-    TCCAnimationTileOverlay *animationOverlay = (TCCAnimationTileOverlay *)self.overlay;
-    
     __weak TCCAnimationTileOverlayRenderer * weakSelf = self;
+    weakSelf.renderedTileZoomLevel = [TCCMapKitHelpers zoomLevelForZoomScale:zoomScale];
+
+    //The overlay can be nil if often or quickly to dealloc the renderer.
```

Comment:
I had trouble understanding this comment. Should this be `The overlay can be nil if called too often or the renderer is deallocated before this code is called.`?